### PR TITLE
build: bump lerna version to 6.x

### PR DESCRIPTION
**Pull request by Yakutoc**
Запланированный переход на **latest** версию `Lerna` - **6.5.1**

:baby_chick: Download canary assets:

[colors--canary.432.4479919276.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.432.4479919276.xml)  
[color_metro_ios-swift--canary.432.4479919276.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.432.4479919276.swift)  
[color_metro_kotlin--canary.432.4479919276.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.432.4479919276.kt)  
[color_metro_react-native--canary.432.4479919276.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.432.4479919276.ts)  
[color_sberHealth_ios-swift--canary.432.4479919276.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.432.4479919276.swift)  
[color_sberHealth_kotlin--canary.432.4479919276.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.432.4479919276.kt)  
[color_sberHealth_react-native--canary.432.4479919276.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.432.4479919276.ts)  
[color_sbermarket_ios-swift--canary.432.4479919276.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.432.4479919276.swift)  
[color_sbermarket_kotlin--canary.432.4479919276.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.432.4479919276.kt)  
[color_sbermarket_react-native--canary.432.4479919276.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.432.4479919276.ts)  
[color_sberprime_ios-swift--canary.432.4479919276.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.432.4479919276.swift)  
[color_sberprime_kotlin--canary.432.4479919276.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.432.4479919276.kt)  
[color_sberprime_react-native--canary.432.4479919276.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.432.4479919276.ts)  
[color_selgros_ios-swift--canary.432.4479919276.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.432.4479919276.swift)  
[color_selgros_kotlin--canary.432.4479919276.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.432.4479919276.kt)  
[color_selgros_react-native--canary.432.4479919276.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.432.4479919276.ts)  
[color_smbusiness_ios-swift--canary.432.4479919276.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.432.4479919276.swift)  
[color_smbusiness_kotlin--canary.432.4479919276.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.432.4479919276.kt)  
[color_smbusiness_react-native--canary.432.4479919276.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.432.4479919276.ts)  
[PlasmaTokensColor--canary.432.4479919276.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.432.4479919276.swift)  
[shadow_sbermarket_react-native--canary.432.4479919276.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.432.4479919276.ts)  
[typo_mage_ios-swift--canary.432.4479919276.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.432.4479919276.swift)  
[typo_mage_kotlin--canary.432.4479919276.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.432.4479919276.kt)  
[typo_mage_react-native--canary.432.4479919276.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.432.4479919276.ts)  
[typo_plasma_ios-swift--canary.432.4479919276.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.432.4479919276.swift)  
[typo_plasma_kotlin--canary.432.4479919276.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.432.4479919276.kt)  
[typo_plasma_react-native--canary.432.4479919276.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.432.4479919276.ts)  
[typo_ruler_ios-swift--canary.432.4479919276.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.432.4479919276.swift)  
[typo_ruler_kotlin--canary.432.4479919276.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.432.4479919276.kt)  
[typo_ruler_react-native--canary.432.4479919276.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.432.4479919276.ts)  
[typo_sage_ios-swift--canary.432.4479919276.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.432.4479919276.swift)  
[typo_sage_kotlin--canary.432.4479919276.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.432.4479919276.kt)  
[typo_sage_react-native--canary.432.4479919276.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.432.4479919276.ts)  
[typo_sbermarket_ios-swift--canary.432.4479919276.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.432.4479919276.swift)  
[typo_sbermarket_kotlin--canary.432.4479919276.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.432.4479919276.kt)  
[typo_sbermarket_react-native--canary.432.4479919276.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.432.4479919276.ts)  
[typo_soulmate_ios-swift--canary.432.4479919276.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.432.4479919276.swift)  
[typo_soulmate_kotlin--canary.432.4479919276.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.432.4479919276.kt)  
[typo_soulmate_react-native--canary.432.4479919276.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.432.4479919276.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.163.1-canary.432.4479919276.0
  npm install @salutejs/plasma-temple@1.144.1-canary.432.4479919276.0
  npm install @salutejs/plasma-ui@1.177.1-canary.432.4479919276.0
  npm install @salutejs/plasma-web@1.188.1-canary.432.4479919276.0
  # or 
  yarn add @salutejs/plasma-b2c@1.163.1-canary.432.4479919276.0
  yarn add @salutejs/plasma-temple@1.144.1-canary.432.4479919276.0
  yarn add @salutejs/plasma-ui@1.177.1-canary.432.4479919276.0
  yarn add @salutejs/plasma-web@1.188.1-canary.432.4479919276.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-432/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-432/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-432/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-432/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-432/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-432/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-432/temple-storybook/

File: package.json [hunk 1]
```diff
@@ -95,7 +95,7 @@
     "http-server": "0.12.3",
     "husky": "^8.0.0",
     "image-loader": "0.0.1",
-    "lerna": "5.6.2",
```
Comment (Yeti-or) [inline package.json:98]: как то очень просто вышло =)

**Review by Yeti-or — APPROVED** [review]

**Comment by Yeti-or** [comment]
мне нравится как уменьшается наши зависимости
Screenshot 2023-03-21 at 10 32 22

File: package.json (hunk 1 above)
Comment (Yakutoc) [reply package.json:98]: @Yeti-or Прошлое обновление тоже просто прошло. 

@neretin-trike У тебя же не было проблем с переходом на 5 версию?

Comment (Yeti-or) [reply package.json:98]: давайте сначала node вольем
а потом отребейзим

Comment (neretin-trike) [reply package.json:98]: да, у меня вроде тоже не было проблем

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-432/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-432/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-432/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-432/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-432/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-432/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-432/temple-storybook/

**Pull request merged**